src/main/java/org/mockito/internal/stubbing/defaultanswers/ReturnsMoreEmptyValues.java

```diff
@@ -61,13 +36,10 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
         Class<?> returnType = invocation.getMethod().getReturnType();
         return returnValueFor(returnType);
     }
-    
+
     Object returnValueFor(Class<?> type) {
         if (type == String.class) {
             return "";
```

Comment:
If this class now only returns an emptry String, why don't we remove this class and move the empty string to `ReturnsEmptyValues`?